torchvision/ops/poolers.py

```diff
@@ -101,11 +120,12 @@ def __init__(self, featmap_names, output_size, sampling_ratio):
         self.map_levels = None
 
     def convert_to_roi_format(self, boxes):
+        # type: (List[Tensor])
         concat_boxes = torch.cat(boxes, dim=0)
         device, dtype = concat_boxes.device, concat_boxes.dtype
         ids = torch.cat(
             [
-                torch.full_like(b[:, :1], i, dtype=dtype, device=device)
+                torch.full((len(b), 1), i, dtype=dtype, device=device)
```

Comment:
This is where the ONNX failure is happening (the size in torch.full() is being saved as a constant).
Does using torch.full_like create issues in scripting?

Reply:
I think this might just be an accidental revert. I'll try changing it, thanks!